scripts-dev/nuke-empty-communities-from-db.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python2
+
+from sys import argv
+from os import system
+import sqlite3
+import psycopg2
+
+if len(argv) < 3 or (argv[1] != "sqlite" and argv[1] != "postgresql") or (argv[1] == "postgresql" and len(argv) < 6):
+    print "Usage:"
+    print "    " + argv[0] + " sqlite <path_to_db_file>"
+    print "or"
+    print "    " + argv[0] + " postgresql <host> <dbname> <user> <password>"
+    quit()
+
+db = None
+if argv[1] == "sqlite":
+    db = sqlite3.connect(argv[2])
+    cur = db.cursor()
+elif argv[1] == "postgresql":
+    db = psycopg2.connect('host=' + argv[2] + ' dbname=' + argv[3] + ' user=' + argv[4] + ' password=' + argv[5])
+    cur = db.cursor()
+else:
+    print "This should be unreachable, report a bug."
+    quit()
+
+with db:
+    cur.execute('SELECT group_id FROM groups;')
```

Comment:
You can extend this query to the following to not need the check in the `for` loop:
```
SELECT g.group_id FROM groups g
    WHERE (SELECT count(*) from group_users u WHERE g.group_id = u.group_id) = 0
```